app/upgrades/v7/constants.go

```diff
@@ -35,10 +38,6 @@ const (
 	stakingMsgCreateValidator           = "/cosmos.staking.v1beta1.MsgCreateValidator"
 	vestingMsgCreateVestingAccount      = "/cosmos.vesting.v1beta1.MsgCreateVestingAccount"
 	ibcMsgTransfer                      = "/ibc.applications.transfer.v1.MsgTransfer"
-	liquidityMsgSwapWithinBatch         = "/tendermint.liquidity.v1beta1.MsgSwapWithinBatch" //#nosec G101 -- This is a false positive
```

Comment:
nit: Why were these constants removed? They are used in `v7/ugprades.go`.

Reply:
Reintroduced